File: API_Banco/NLP-Server/server/models/classify.ts
import * as mongoose from 'mongoose';

const classifySchema = new mongoose.Schema({
    classifyName: String,
    desClassify: String,
    classifyToken: String,
    idUser: mongoose.Schema.Types.ObjectId,
    dtInsert: Date,
    dtUpdate: Date
});

const Classify = mongoose.model('Classify', classifySchema);

export default Classify;

File: Trainer/classify.py
from mongoengine import *

class Classifies(Document):
    _id = ObjectIdField(required=False)
    classifyName = StringField(required=True)
    desClassify = StringField(required=True)
    dtInsert = DateTimeField(required=True)
    dtUpdate = DateTimeField(required=False)
    idUser = ObjectIdField(required=True)
    classifyToken = StringField(required=True)
    version = IntField(db_field="__v", required=False)  